backend/src/main/java/atlantbh/restaurants/models/Restaurant.java

```diff
@@ -0,0 +1,157 @@
+package atlantbh.restaurants.models;
+
+import javax.persistence.*;
```

Comment:
avoid import all, if you don't really need it.

Reply:
Fixed 8c841a20e489a31c66d177c8682339f9d4f19de0